clang/test/Driver/Inputs/libsycl.ll

```diff
@@ -0,0 +1,36 @@
+; ModuleID = 'libsycl.cpp'
+source_filename = "libsycl.cpp"
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64-G1"
+target triple = "spirv64"
+
+; Function Attrs: mustprogress nofree noinline norecurse nosync nounwind willreturn memory(none)
+define spir_func noundef range(i32 -2147483643, -2147483648) i32 @_Z9lib_func1i(i32 noundef %a) local_unnamed_addr #0 {
+entry:
+  %add = add nsw i32 %a, 5
+  ret i32 %add
+}
```

Comment:
I would prefer to have simpler code for the library (and tests):
1. Unmingled names.
2. No unnecessary attributes.
3. Less code. Do we need 4 library functions? I would leave just one:

```llvm
define spir_func i32 addFive(i32 %x) {
  %res = add i32 %x, 5
  ret i32 %res
}
```

It's easier to maintain and read.